commit 987bf5bbeb25e846617d7bc1ee9b78f049dc0d60
Author: Rafal Luzynski <digitalfreak@lingonborough.com>
Date:   Tue Mar 13 02:28:42 2018 +0100

    gdatetime: Add missing #define WEEKDAY_FULL_IS_LOCALE
    
    One more #define WEEKDAY_FULL_IS_LOCALE was missing from the commit
    12f11090dc1b6062f4a493d79b382714ebbdc413.
    
    https://bugzilla.gnome.org/show_bug.cgi?id=793578
---
 glib/gdatetime.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/glib/gdatetime.c b/glib/gdatetime.c
index 2eda46664..ba8503813 100644
--- a/glib/gdatetime.c
+++ b/glib/gdatetime.c
@@ -224,6 +224,7 @@ static const gint month_item[2][12] =
 #define WEEKDAY_ABBR(d)       (get_weekday_name_abbr (g_date_time_get_day_of_week (d)))
 #define WEEKDAY_ABBR_IS_LOCALE FALSE
 #define WEEKDAY_FULL(d)       (get_weekday_name (g_date_time_get_day_of_week (d)))
+#define WEEKDAY_FULL_IS_LOCALE FALSE
 /* We don't yet know if nl_langinfo (MON_n) returns standalone or complete-date
  * format forms but if nl_langinfo (ALTMON_n) is not supported then we will
  * have to use MONTH_FULL as standalone.  The same if nl_langinfo () does not
